arbitrum-docs/launch-orbit-chain/bold-adoption-for-orbit-chains.mdx

```diff
@@ -0,0 +1,106 @@
+
+Adopting <a data-quicklook-from="bold">Arbitrum BoLD</a> brings new security benefits to Orbit chains that are independent of whether or not validation is permissionless. These new security benefits include improved resistance to delay attacks and increased censorship resistance for `L3s`. Because of these new security benefits, we strongly recommend teams to upgrade their Orbit chains to use Arbitrum BoLD (for the security benefits) but **keep validation permissioned**. Below is a quick breakdown of the impacts of permissioned BoLD vs. permissionless BoLD on your Orbit chain:
+
+
+#### Permissioned BoLD on an Orbit chain
+
+##### Benefits                                                                                                           
+    - Improved resistance to delay attacks                                                                               
+    - Increased censorship resistance for L3s                                                                            
+    - Use of any `ERC-20` as the bonding token                                                                           
+    - Be on the latest iteration of Arbitrum technology, making future security & feature updates much easier to pick up 
+    - Less engineering overhead and infrastructure costs                                                                 
+
+##### Caveats 
+    - Permissioned validation: only entities on allowlist can resolve disputes
+
+#### Permissionless BoLD on an Orbit chain
+
+##### Benefits                                                                                                           
+    - Improved resistance to delay attacks                                                                               
+    - Increased censorship resistance for L3s                                                                            
+    - Use of any `ERC-20` as the bonding token                                                                           
+    - Be on the latest iteration of Arbitrum technology, making future security & feature updates much easier to pick up 
+    - Less engineering overhead and infrastructure costs                                                                 
+    - More trust in the system's integrity and fairness                                                                  
+    - Creates a more open ecosystem for third-party developers and users                                                 
+
+##### Caveats 
+    - Resource exhaustion attacks: challenges require computing and financial resources to participate in, and an attacker could have significantly more resources than the defenders
+    - Liveness impact: each challenge takes time to resolve, and so if the cost of a challenge is cheap, then an attacker can open challenges and prevent the progress of the chain at a low price 
```

Comment:
```suggestion
    - Liveness impact: if the cost of a challenge is cheap, then an attacker can cheaply open a challenge and prevent the progress of the chain for as long as they can censor honest moves
```

censorship of honest moves is also required to delay progress

Reply:
Added!